Code/GraphMol/MolOps.h

```diff
@@ -212,25 +234,34 @@ RDKIT_GRAPHMOL_EXPORT unsigned int getMolFragsWithQuery(
        - the caller is responsible for <tt>delete</tt>ing the pointer this
    returns.
  */
-RDKIT_GRAPHMOL_EXPORT ROMol *addHs(const ROMol &mol, bool explicitOnly = false,
-                                   bool addCoords = false,
-                                   const UINT_VECT *onlyOnAtoms = nullptr,
-                                   bool addResidueInfo = false);
+inline ROMol *addHs(const ROMol &mol, bool explicitOnly = false,
+                    bool addCoords = false,
+                    const UINT_VECT *onlyOnAtoms = nullptr,
+                    bool addResidueInfo = false) {
+  AddHsParameters ps{explicitOnly, addCoords, addResidueInfo};
+  std::unique_ptr<RWMol> res{new RWMol(mol)};
+  addHs(res.get(), ps, onlyOnAtoms);
```

Comment:
My bad, this should have stayed as:
```suggestion
  addHs(*res, ps, onlyOnAtoms);
```

Reply:
> My bad, this should have stayed as:

yeah, well, I could have noticed that too.
anyway... it's fixed now